Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -72,15 +88,21 @@ def __init__( self, ceUniqueID ):
     self.outputURL = 'gsiftp://localhost'
     self.gridEnv = ''
     self.proxyRenewal = 0
+    self.extraSubmitString = self.ceParameters.get('ExtraSubmitString', '').decode('string_escape')
+
+    ## see note on getCondorLogFile, why we can only use the global setting
+    self.workingDirectory = gConfig.getValue( "Resources/Computing/HTCondorCE/WorkingDirectory",
```

Comment:
The pilot logging needs certainly a complete rework. This is what is supposed to have been done in the already committed PRs ( v6r17 ? ). When and if this will be done and will be working, this log extracting mechanism will not be needed any more. So, let's keep it as it s for the moment.